packages/pwa/app/components/drawer-menu/index.jsx

```diff
@@ -267,7 +268,7 @@ const DrawerMenu = ({isOpen, onClose = noop, onLogoClick = noop, root}) => {
                                     <LocaleSelector
                                         {...styles.localeSelector}
                                         selectedLocale={intl.locale}
-                                        locales={supportedLocaleIds}
+                                        locales={getSupportedLocalesIds(site.l10n.supportedLocales)}
```

Comment:
It looks like `getSupportedLocalesIds(site.l10n.supportedLocales)` is being assigned to `supportedLocaleIds` above, can you use that variable?

Reply:
Good catch, it is resolved